upstairs/src/lib.rs

```diff
@@ -1392,9 +1406,9 @@ where
                          * must not get a message.
                          */
                         match op {
-                            Message::ExtentRepair(rep_id, _, _, _, ref dest) => {
+                            Message::ExtentRepair { repair_id, extent_id: _, source_client_id: _, source_repair_address: _, ref dest_clients} => {
```

Comment:
I think `cargo fmt` failed to handle this line

Reply:
fixed in 2559476